tests/file_storage/tests.py

```diff
@@ -842,6 +847,48 @@ def test_stringio(self):
             self.assertEqual(f.read(), b'content')
 
 
+class FieldCallableFileStorageTests(TestCase):
+    def setUp(self):
+        self.temp_storage_location = tempfile.mkdtemp(suffix='filefield_callable_storage')
+
+    def tearDown(self):
+        shutil.rmtree(self.temp_storage_location)
+
+    def test_callable_base_class_error_raises(self):
+        class NotStorage:
+            pass
+        msg = "FileField.storage must be a subclass/instance of django.core.files.storage.Storage"
+        with self.assertRaisesMessage(ImproperlyConfigured, msg):
+            FileField(storage=NotStorage)
+            FileField(storage=None)
+            FileField(storage=str)
+            FileField(storage=list)
+            FileField(storage=set)
```

Comment:
Would this block be better using [`subTest()`](https://docs.python.org/3.7/library/unittest.html#unittest.TestCase.subTest)?